pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "PowerSploit";
+  version = "v3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = finalAttrs.version;
+    sha256 = "sha256-cFj4IwDyiwQDSYLCcloMQ83T8MOtMcKsyMilfrZOyHs=";
```

It's a SRI hash, thus `hash` is the preference nowadays.

```suggestion
    hash = "sha256-cFj4IwDyiwQDSYLCcloMQ83T8MOtMcKsyMilfrZOyHs=";
```